wasm2c/examples/segue/dhrystone_src/dhry.h

```diff
@@ -0,0 +1,306 @@
+/*
```

Comment:
Perhaps add a readme that links to where these files were copied from?

Reply:
Will fix